extension/surveys/survey-generator.js

```diff
@@ -571,3 +571,45 @@ function getFormValues(questionArr, form) {
   }
   return responses;
 }
+
+/**
+ * Checks whether a form has any responses or text in it.
+ * @param {object} form The form to check.
+ * @returns {boolean} True if the form has content.
+ */
+function formHasContent(form) {
+  for (var i = 0; i < form.elements.length; i++) {
+    var element = form.elements[i];
+    switch (element.type) {
+      case 'checkbox':
+      case 'radio':
+        if (element.checked)
+          return true;
+        break;
```

Comment:
We do need the break, since only one path of the if-else returns. The break is required whenever we hit the else case.